osrf/vrx
# [Garden] Port scan, dock, deliver - Part 2

This pull request partially ports the ScanDockScoringPlugin from Gazebo classic. In particular, it ports the functionality related with the dock checker.

### How to test it?

* Launch the scan_dock_deliver_task world:
```
ros2 launch vrx_gz competition.launch.py world:=scan_dock_deliver_task
```

When the simulation transitions to the `ready` state (~10secs), you should see how the placards update their symbols to the ones specified in SDF (yellow circle, blue triangle and red rectangle).

* On a new terminal launch the joystick teleop node:
```
ros2 launch vrx_gz usv_joy_teleop.py
```

* On a new terminal check the `/vrx/task/info` topic:
```
ros2 topic  echo /vrx/task/info
```

* Now, teleop the WAM-V to a bay. A successful docking maneuver consists on staying for around `<min_dock_time>` (10 seconds) in a docking bay, and then, leave. Verify that you get `15` points for docking in any bay and `5` extra points if you dock in the correct bay (the central one). Keep also in mind that the simulation will terminate after any successful docking.

**Bonus**: I reduced the sensitivity of the joystick teleoperation.
